src/components/movies/MoviesNewForm.js

```diff
@@ -0,0 +1,117 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { createMovie } from "../../api/fetch";
+import "./MoviesForm.css";
+
+export default function MoviesNewForm() {
+  const [movie, setMovie] = useState({
+    type: "",
+    title: "",
+    country: "",
+    dateAdded: "",
+    description: "",
+    duration: "",
+    listedIn: "",
+    rating: "",
+    releaseYear: "",
+  });
+
+  const navigate = useNavigate();
+  function handleSubmit(event) {
+    event.preventDefault();
+    createMovie(movie)
+    .then((response) => {
+      navigate(`/movies/${response.id}}`);
```

Comment:
there's an extra closing curly bracket that's messing things up

Reply:
Thanks for catching that Angie, I just got done fixing it.